### DISCUSSION TITLE
GUI Not Loading on Windows Server or macOS Client

### DISCUSSION THREAD

**Discussion by rememberthis9000** [apohl79/audiogridder#832] [Help] [answered]
**Plugin OS**: macOS 12.0 
**Plugin DAW**: Logic Pro 10.7.1
**Server OS**: Windows 11
**Plugin and Server Version**: 1.2.0b6

Only Generic Editor appears:

<img width="677" alt="Screen Shot 2021-11-28 at 5 40 54 PM" src="https://user-images.githubusercontent.com/95193465/143791091-968caf09-4097-439b-b32a-d691d86022eb.png">

Windows Server does not load plugin GUI either:

<img width="993" alt="Screen Shot 2021-11-28 at 5 42 51 PM" src="https://user-images.githubusercontent.com/95193465/143791139-2775ee96-ade1-4f44-87e1-9bc7bb24d73e.png">

I have only tried it with a couple of plugins so far. Server and client FX log attached.
[AudioGridderServer_2021-11-28_17-05-22.log](https://github.com/apohl79/audiogridder/files/7614664/AudioGridderServer_2021-11-28_17-05-22.log)
[AudioGridderPlugin_2021-11-28_17-40-45.log](https://github.com/apohl79/audiogridder/files/7614671/AudioGridderPlugin_2021-11-28_17-40-45.log)

Any help would be appreciated!

**Edit**: Tried version 1.1.0 and multiple well-known plugins (e.g. Waves, FabFilter, NadIR) with the same issues, so it's not the plugins.

**Comment by rememberthis9000** [comment]
Bump

**Comment by CrossGridder** [comment]
What if you check the task manager? How many audiogridderserver.exe processes are there?

**Reply by rememberthis9000** [reply]
Only one process. I've also tried reinstalling on both the client and server, and I just tried using a different server (a Windows virtual machine) to no avail.

**Reply by CrossGridder** [reply]
What if you change Audiogridderserver.exe compatibility to windows 7 or 10?

**Reply by rememberthis9000** [reply]
I tried that, no go. Good idea, though! I also tried running the server in Windows 10 and installing version 1.1.1 on the server and client with the same results.

**Comment by Henri75** [comment]
First guess : graphics driver issue, or may be a subsystem like DirectX or OpenGL ?

**Comment by Merx08** [comment]
I am just now having the same issue, it was showing up just fine. I then rescanned in some new plugins and boom.. no longer loading the GUI.. what a bummer. I have done some much troubleshooting and even updated to the latest beta to no avail. It would be great if we knew the root problem to this.

**Comment by DeeSaR1** [comment, marked as answer]
Uncheck "Generic Editor" in the settings of the client plugin. (Was enabled by default in my installation too and resulted in not displaying the gui (wether on the server, nor on the client))

On the server side make sure in settings the "Scree Capture Mode" is set to "FFmpeg (webp)"

**Reply by rememberthis9000** [reply]
That worked. Simple oversight.. Thank you!
